DirectEmployers/MyJobs
# [PD-3846] search field larger than container in windows

## What's this PR do?
PR keeps the search box inside of it's container

## Screenshots (if appropriate)
Refer to: REFER to: https://directemployersfoundation.atlassian.net/browse/PD-3846

## Where should the reviewer start?
Start inside any of the analytics apps using a WINDOWS machine or VM if you are on OSX

## How should this be manually tested?
Pull down the repository. Go into any analytics application. The problems are only persisting in windows and not OSX. So you will need a VM if you are running OSX to test. Make sure the search box container is contained and not running longer than container

## Any background context you want to provide?
REFER to: https://directemployersfoundation.atlassian.net/browse/PD-3846
For Comments and testing purposes

## PR Checklist

### Front-End


### Separation of Concerns


### Tests


### Migrations
